dev_tools/check_useless_exclude_paths_hooks.py

```diff
@@ -101,10 +112,22 @@ def have_non_existent_paths_or_duplicates(hooks_list: list[Any]) -> bool:
     return bool(non_existing_paths or duplicates)
 
 
+def print_excluded_files_count(hooks_list: list[Any]) -> None:
+    total_excluded = sum(hook.count_excluded_files() for hook in hooks_list)
+    print(f"Total number of excluded files: {total_excluded}")
+
+    for hook in hooks_list:
+        excluded_count = hook.count_excluded_files()
+        if excluded_count > 0:
+            print(f"Hook {hook.id}: Files excluded: {excluded_count}")
```

Comment:
Just making this a bit nicer to read

```suggestion
            print(f"{excluded_count}\tfiles excluded from hook {hook.id}")
```